ipaserver/install/adtrustinstance.py

```diff
@@ -130,6 +130,15 @@ def map_Guests_to_nobody():
     logger.debug("Map BUILTIN\\Guests to a group 'nobody'")
     ipautil.run(args, env=env, raiseonerr=False, capture_error=True)
 
+
+def get_idmap_range(realm):
+    range = api.Command.idrange_show('%s_id_range' % realm)['result']
+    range_start = int(range['ipabaseid'][0])
```

Comment:
Is the ``int()`` call actually necessary? The API should return a proper integer here.

Reply:
Yes, it needs it here because it is a simple dictionary:
```
>>> range = api.Command.idrange_show('%s_id_range' % api.env.realm)['result']
>>> range
{'ipaidrangesize': ['200000'], 'cn': ['EXAMPLE.COM_id_range'], 'ipabaserid': ['1000'], 'ipasecondarybaserid': ['100000000'], 'ipabaseid': ['1536000000'], 'iparangetype': ['local domain range'], 'iparangetyperaw': ['ipa-local'], 'dn': ipapython.dn.DN('cn=EXAMPLE.COM_id_range,cn=ranges,cn=etc,dc=example,dc=com')}
>>> range.get('ipabaseid')
['1536000000']
>>> range.__class__
<class 'dict'>
```

Reply:
Ah, the values are strings, not ints. I was expecting ints here.